crates/bevy_text/src/text.rs

```diff
@@ -323,11 +328,38 @@ impl Default for TextFont {
         Self {
             font: Default::default(),
             font_size: 20.0,
+            line_height: LineHeight::default(),
             font_smoothing: Default::default(),
         }
     }
 }
 
+/// Specifies the height of each line of text for `Text` and `Text2d`
+///
+/// Default is 1.2x the font size
+#[derive(Debug, Clone, Copy, Reflect)]
+pub enum LineHeight {
+    /// Set line height to a specific number of pixels
+    Pixels(f32),
+    /// Set line height to a multiple of the font size
+    Scalar(f32),
+}
+
+impl LineHeight {
+    pub(crate) fn eval(self, font_size: f32) -> f32 {
+        match self {
+            LineHeight::Pixels(px) => px,
+            LineHeight::Scalar(scale) => scale * font_size,
+        }
+    }
+}
+
+impl Default for LineHeight {
+    fn default() -> Self {
+        LineHeight::Scalar(1.2)
```

Comment:
To make it easier to expand the feature set in the future (by propagating inherited line height across a hierarchy etc) without having to do a breaking change, would it make sense to have a separate `LineHeight::Auto` variant as the default?

Reply:
What would that do?